templates/partials/_snap-details-video-layout.html

```diff
@@ -4,7 +4,7 @@
    see https://github.com/canonical-web-and-design/vanilla-framework/issues/2448 #}
   <div class="col-10 u-align--center">
     {% for video in videos %}
-      <div class="js-video-slide" data-video-type="{{video.type}}" data-video-url="{{video.url}}" data-video-id="{{video.id}}">
+      <div class="js-video-slide u-embedded-media" data-video-type="{{video.type}}" data-video-url="{{video.url}}" data-video-id="{{video.id}}">
```

Comment:
The videos are embedded in a loop. Therefore, I assume there could be multiple with the same `id`?

Reply:
same as above